.github/PULL_REQUEST_TEMPLATE/production.md

```diff
@@ -30,6 +30,7 @@ I have...
 * [ ] Updated the relevant documentation or specification
 * [ ] Reviewed "Files changed" and left comments if necessary <!-- relevant if the changes are not obvious  -->
 * [ ] Confirmed all CI checks have passed
+* [ ] If this PR is library API breaking, bump the go.mod version string of the repo, and follow through on a new major release for both the consumer and provider
```

Comment:
Do we want to do this for every API breaking PR?

Reply:
If we bump the major version of ICS, golang requires us to bump the version string. We can redefine what constitutes a major release, but that'd be steering away from how IBC does versioning

Reply:
My comment is more generally about the mapping between a single PR and a release. Yes, a release that contains a PR with library API breaking changes should have the major version bumped. But not every PR with library API breaking changes should result in a new release.